bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/handler/TeslaVehicleHandler.java

```diff
@@ -991,7 +991,8 @@ protected boolean establishEventStream() {
                     eventClient = clientBuilder.build()
                             .register(new Authenticator((String) getConfig().get(CONFIG_USERNAME), vehicle.tokens[0]));
                     eventTarget = eventClient.target(URI_EVENT).path(vehicle.vehicle_id + "/").queryParam("values",
-                            StringUtils.join(EventKeys.values(), ',', 1, EventKeys.values().length));
+                            Arrays.asList(EventKeys.values()).stream().map(Enum::toString)
+                                    .collect(Collectors.joining(",")));
```

Comment:
Is this really the same? Or do we have to ignore the first element?

Reply:
Seems like we need to `skip()` one.

Reply:
Thanks. I added `skip(1)`.